tests/test_graphical_units/test_creation.py

```diff
@@ -5,10 +5,10 @@
 from ..utils.GraphicalUnitTester import GraphicalUnitTester
 
 
-class ShowCreationTest(Scene):
+class CreateTest(Scene):
```

Comment:
Renaming this class makes it necessary to also rename the corresponding test output file, `tests/control_data/graphical_units_data/creation/ShowCreationTest.npz` needs to be renamed to `CreateTest.npz`.

Reply:
Thanks a lot for the suggestion, @behackl! I'll run the tests locally and see whether any other changes may be needed. I didn't do a good job find-and-replacing the `ShowCreation` instances in one go and have frankly made a mess with committing so much. I plan to re-upfetch the pre-PR code and rename the instances in a more organized manner :)